### DISCUSSION TITLE
Can we install new routes after listen is called

### DISCUSSION THREAD

**Discussion by luxapana** [uNetworking/uWebSockets#1291] [Q&A]
We have a requirement to add new routes to an already running server. Is this supported?

**Comment by e3dio** [comment]
Yes you can add new routes, but you can't replace or remove routes. It seems easy to add a replace/remove feature though if needed

**Reply by luxapana** [reply]
Are there any plans to implement this functionality in the near future? (remove/modify an existing route). Thanks.
